pkg/nas/utils.go

```diff
@@ -108,7 +108,13 @@ func doMount(mounter mountutils.Interface, opt *Options, targetPath, volumeId, p
 			mountFstype = "alinas"
 			// must enable tls when using accesspoint
 			combinedOptions = addTLSMountOptions(combinedOptions)
-		} else {
+		}
+		// Enable compatibility for BMCPFS VPC mount points using the NAS Driver, to support the same usage in ECI.
+		if strings.HasSuffix(opt.Server, "cpfs.aliyuncs.com") && opt.MountProtocol == "efc" {
```

Comment:
Are there alternative methods to avoid modifying mount options based on domain suffixes, especially for custom domains?